test/coordinator-test.js

```diff
@@ -0,0 +1,19 @@
+import { assert } from 'chai';
+import os from 'os';
+import sinon from 'sinon';
+import { getWorkerCount } from '../lib/coordinator';
+
+describe('coordinator', () => {
+  it('uses the correct number of cpus', () => {
+    const sandbox = sinon.sandbox.create();
+    sandbox.stub(os, 'cpus').returns([{}, {}, {}, {}, {}]);
```

Comment:
I think that a more important principle is that the relationship between values should be expressed in code, and not be implicit, even in tests.